### PR TITLE
VZ-7875: Pipeline to download source files

### PR DESCRIPTION
**Pull request by bhatpmk**
This change adds a new Jenkinsfile to download the source from various repositories defined in an input properties file and creates an archive.

### CODE REVIEW

File: release/builds/JenkinsfilePRT [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    parameters {
+        string (description: 'The release branch', name: 'RELEASE_BRANCH', defaultValue: 'NONE', trim: true)
+        string (description: 'The source commit for the release (required for full release)', name: 'COMMIT_TO_USE', defaultValue: 'NONE', trim: true )
+        string (description: 'The release version (major.minor.patch format, e.g. 1.5.0)', name: 'RELEASE_VERSION', defaultValue: 'NONE', trim: true)
+    }
+
+    environment {
+        OCR_CREDS = credentials('ocr-pull-and-push-account')
```
Comment (agv426) [inline release/builds/JenkinsfilePRT:27]: Remove these unused creds, looks like none of the ones in this block are used (at least directly). OCR_CREDS, NETRC_FILE, DOCKER_CREDS, DOCKER_REPO

File: release/builds/JenkinsfilePRT [hunk 2]
```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
```
Comment (agv426) [inline release/builds/JenkinsfilePRT:1]: Thinking we should use a better name for this Jenkinsfile, maybe JenkinsfileSourceArchive or something similar

If we ever automate the PRT process itself that Dave needs to follow, then we may call that JenkinsfilePRT,

Comment (bhatpmk) [reply release/builds/JenkinsfilePRT]: Done

File: release/builds/JenkinsfilePRT (hunk 1 above)
Comment (bhatpmk) [reply release/builds/JenkinsfilePRT]: Done

**Review by agv426 — APPROVED** [review]

**Pull request merged**